src/utils/enums.ts

```diff
@@ -4,3 +4,32 @@ export const enum Keyboard {
   ARROW_UP = 'ArrowUp',
   TAB = 'Tab',
 }
+
```

Comment:
Se não for algo reutilizável para mais componentes, pode colocar em um arquivo dentro de avatar.